ocaml/squeezed/lib/squeeze.ml

```diff
@@ -649,11 +649,9 @@ let change_host_free_memory ?fistpoints io required_mem_kib success_condition =
           []
     in
     let new_target_direction domain =
-      match List.assoc_opt domain.domid new_targets with
-      | None ->
-          string_of_direction None
-      | Some t ->
-          string_of_direction (direction_of_int64 domain.target_kib t)
+      List.assoc_opt domain.domid new_targets
+      |> Option.value ~default:domain.target_kib
+      |> Int64.to_string
```

Comment:
```
      |> direction_of_int64
      |> string_of_direction
```

Reply:
If we would keep this way, we should use `Option.bind`.  But using match is more plain，it remain unchanged.